apps/sms/test/marionette/mocks/mock_navigator_moz_mobile_message.js

```diff
@@ -3,6 +3,7 @@
 
 const Cu = Components.utils;
 Cu.import('resource://gre/modules/Services.jsm');
+Cu.import('resource://gre/modules/Promise.jsm');
```

Comment:
Looks like now it works without ```Cu.import('resource://gre/modules/Promise.jsm');```, heh, just 1.5 months and two issues have gone :D